src/types.ts

```diff
@@ -0,0 +1,15 @@
+export interface VictimDetail {
+  id: string;
+  lat: string;
+  lon: string;
+  medicalNeeded: boolean;
+  numberOfPeople: number;
+  victimName: string;
+  victimPhoneNumber: string;
+  timeStamp: number;
+  status: string;
+}
+
+export default interface SearchInput {
```

Comment:
do we need it to be default export?

Reply:
This is only here to satisfy the old code. This will not be needed after the refactor code is merged.